src/garage/experiment/deterministic.py

```diff
@@ -15,13 +16,22 @@ def set_seed(seed):
 
     """
     seed %= 4294967294
+    # pylint: disable=global-statement
     global seed_
     seed_ = seed
     random.seed(seed)
     np.random.seed(seed)
     if 'tensorflow' in sys.modules:
-        import tensorflow as tf
+        import tensorflow as tf  # pylint: disable=import-outside-toplevel
         tf.compat.v1.set_random_seed(seed)
+    if 'torch' in sys.modules:
+        warnings.warn(
+            'Enabeling deterministic mode in PyTorch can have a performance \
```

Comment:
python joins strings between () implicitly, even if they are on different lines.

e.g.
```python
warnings.warm(
    'foo'
    'bar')
```
will result in warning text `foobar`